### PR TITLE
API start point

### PR DESCRIPTION
**Pull request by KMayne**
# What?
- The plumbing for the API endpoints with the endpoints returning mock data
- A method for specifying a local configuration e.g. local database config that is gitignored

# Why?
Make it a lot easier to develop the backend API.

### CODE REVIEW

**Comment by KMayne** [comment]
I think the CI failure is because the build isn't fully configured yet
@mauriceyap Is this right?

File: apps/etheroscope_db/lib/etheroscope_db/application.ex [hunk 1]
```diff
@@ -8,22 +8,7 @@ defmodule EtheroscopeDB.Application do
   def start(_type, _args) do
     children = []
 
-    create_database()
```
Comment (Thopiax) [inline apps/etheroscope_db/lib/etheroscope_db/application.ex:11]: Where did this end up going?

File: apps/etheroscope_web/lib/etheroscope_web/router.ex [hunk 2]
```diff
@@ -8,6 +8,7 @@ defmodule EtheroscopeWeb.Router do
   scope "/", EtheroscopeWeb do
     pipe_through :api
 
-    get "/history/:contract_address/:method", ContractController, :history
+    get "/contracts/:contract_address/", ContractController, :contract
+    get "/contracts/:contract_address/history/:variable", ContractController, :history
```
Comment (Thopiax) [inline apps/etheroscope_web/lib/etheroscope_web/router.ex]: I would put history before contract_address.

**Review by Thopiax — CHANGES_REQUESTED** [review]
Otherwise it's good 👍

File: apps/etheroscope_web/lib/etheroscope_web/router.ex (hunk 2 above)
Comment (KMayne) [reply apps/etheroscope_web/lib/etheroscope_web/router.ex]: That's not as restful IMO

File: apps/etheroscope_db/lib/etheroscope_db/application.ex (hunk 1 above)
Comment (KMayne) [reply apps/etheroscope_db/lib/etheroscope_db/application.ex:11]: See message here https://github.com/Etheroscope/backend-phoenix/pull/5/commits/460d5ee6cb89944b6759a2a848677bc4157fcfd5

File: apps/etheroscope_web/lib/etheroscope_web/router.ex (hunk 2 above)
Comment (mauriceyap) [reply apps/etheroscope_web/lib/etheroscope_web/router.ex]: suggest `/history` gives histories for all variables
suggest `/history?variables=[var1,var2]` gives histories for var 1 and var 2

Comment (KMayne) [reply apps/etheroscope_web/lib/etheroscope_web/router.ex]: Sounds good

**Comment by KMayne** [comment]
@Thopiax Thoughts on the changes?

**Review by Thopiax — APPROVED** [review]

**Pull request merged**